pkg/cmd/cli/cmd/deploy.go

```diff
@@ -26,6 +26,12 @@ NOTE: This command is still under active development and is subject to change.`
 	deploy_example = `  // Display the latest deployment for the 'database' deployment config
   $ %[1]s deploy database
 
+  // List all deployments for the 'database' deployment config
+  $ %[1]s deploy database -L
```

Comment:
Ok, let me cook something around `describe`. What do you think about displaying the complete history, but only the latest 3 deployments in history in full details (like the latest one is displayed today), and all others summarized? Or let's just display the full history with details (I didn't check yet about performance, not sure if every deployment would require an additional api call, etc)?